notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -267,7 +267,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    PROJECT_ID = ""
</pre><p>This should be:</p><p> </p><pre class="ql-syntax" spellcheck="false">PROJECT_ID = "[YOUR-PROJECT-ID]"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>